app/utils/PagerDutyHelper.scala

```diff
@@ -31,6 +31,7 @@ object PagerDutyHelper {
     final val FAILED_TO_PARSE_REQUEST_TO_MODEL = Value
     final val FAILED_TO_INSERT_FILE_NOTIFICATION = Value
     final val NOTIFICATION_SET_TO_PERMANENT_FAILURE = Value
+    final val NOTIFICATION_SET_TO_NOT_PROCESSED_PENDING_RETRY = Value
```

Comment:
Need an alert config PR otherwise we'd never know this was being thrown

Reply:
https://github.com/hmrc/alert-config/pull/2919 Created